source/form.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html class="page" lang="ru">
+
+<head>
+  <meta charset="utf-8">
+  <title>HTML Academy: Мишка</title>
+</head>
+
+<body class="page-body">
+  <header class="page-header">
+    <h1 class="page-header__title">Вязание на заказ</h1>
+    <a class="page-header__logo" href="../source/index.html">
+      <img src="href=https://via.placeholder.com/150x150" width="85" height="34" alt="Логотип сайта">
+    </a>
+    <nav class="main-nav">
+      <div class="main-nav__wrapper">
+        <button class="main-nav__toggle" type="button"><span class="visually-hidden">Открыть меню</span></button>
+        <ul class="main-nav__list site-list">
+          <li class="site-list__item">
+            <a href="../source/catalog.html">Каталог товаров</a>
+          </li>
+          <li class="site-list__item">
+            <a href="#">Вязание на заказ</a>
+          </li>
+          <li class="site-list__item">
+            <a href="#">Поиск по сайту</a>
+          </li>
+          <li class="site-list__item">
+            <a href="#">Корзина: пока пуста</a>
+          </li>
+        </ul>
+      </div>
+    </nav>
+  </header>
+  <main class="page-main">
+    <h1 class="visually-hidden">Форма заказа</h1>
+    <p class="page-main__description">Мы будем рады воплотить в жизнь ваши пожелания! Заполните простую форму заказа и
+      мы свяжемся с вами, чтобы уточнить детали.</p>
+    <form class="page-main__product-form product-form" action="https://echo.htmlacademy.ru" method="POST">
+      <fieldset class="product-form__type">
+        <legend class="product-form__fields-name">Тип</legend>
+        <ul class="product-form_type-list type-list">
+          <li class="type-list__item">
+            <input type="radio" name="product-type" id="filter-interior-accessories" value="interior-accessories"
+              checked>
+            <label for="filter-interior-accessories">Аксессуар для интерьера</label>
+          </li>
+          <li class="type-list__item">
+            <input type="radio" name="product-type" id="filter-kids-toy" value="kids-toy">
+            <label for="filter-kids-toy">Детская игрушка</label>
+          </li>
+        </ul>
+      </fieldset>
+      <fieldset class="product-form__color">
+        <legend class="product-form__fields-name">Цвет</legend>
+        <ul class="product-form_color-list color-list">
```

Comment:
`product-form_color-list` - элемент неправильно оформлен
```suggestion
        <ul class="product-form__color-list color-list">
```